server/apps/archive/archive_composite.py

```diff
@@ -61,6 +63,53 @@ def get_item_ref(item):
 
 class PackageService():
```

Comment:
Nop, it's not a resource service, more like a helper service. You can't get it through `get_resource_service`